src/meili/map_matching.cc

```diff
@@ -46,11 +51,12 @@ State::route(const std::vector<const State*>& states,
   }
 
   // Route
-  labelset_ = std::make_shared<LabelSet>(std::ceil(max_route_distance));
+  labelset_ = std::make_shared<LabelSet>(max_route_distance);
```

Comment:
i did that in a previous commit, but then i reverted it since i moved the check of the max time out of the labelset. how would i make use of it if i pass in max_route_distance?